test/test_heuristics.rb

```diff
@@ -317,6 +321,13 @@ def test_nl_by_heuristics
     })
   end
 
+  def test_p_by_heuristics
+    assert_heuristics({
+      "Gnuplot" => all_fixtures("Gnuplot", "*"),
+      "OpenEdge ABL" => all_fixtures("OpenEdge ABL", "*")
+    }, alt_name="test.p")
```

Comment:
As I understand, the `alt_name` is meant to ensure all files actually go through the heuristic for the `.p` extension, regardless of their extension. Could you add a comment in your code here, just to clarify why we're doing this?

Reply:
Done. In the `assert_heuristics` function.